i18n/es.jsonp.js

```diff
@@ -246,14 +247,14 @@ document.localeJson = {
-      "about-contentinjection-para4": "<b>Modo JQuery</b>: Este modo ahora está obsoleto (y de hecho no requiere JQuery). Se mantiene para la compatibilidad con los navegadores antiguos o los frameworks que no pueden ejecutar Service Workers. El modo tiene limitaciones que significan que sólo se puede mostrar contenido estático, como el que se encuentra en los archivos de Wikipedia / WikiMedia y (por ahora) Stackexchange. Es una forma de insertar artículos extraídos del ZIM en el DOM (documento del navegador) inyectando el contenido en un iframe o pestaña. A continuación, utilizamos métodos DOM nativos, o a veces JQuery, para analizar el HTML del artículo e insertar los activos necesarios (imágenes, hojas de estilo, escuchadores de eventos para los hipervínculos, etc.). En los dispositivos antiguos, el recorrido del DOM puede ser lento, pero se compensa porque no extraemos ni ejecutamos activos JavaScript (lo que sería técnicamente extremadamente complicado). Como resultado, para los archivos WikiMedia este modo suele ser bastante rápido. Por otro lado, los ZIM que tienen una interfaz de usuario dinámica propietaria (como Gutenberg o TED Talks) sólo se admiten parcialmente en este modo: la interfaz de usuario no funciona, pero los artículos se pueden buscar y cargar desde la barra de búsqueda.",
+      "about-contentinjection-para4": "<b>Modo Seguro</b>: Este modo impide que los scripts adjuntos se ejecuten en el iframe, por lo que resulta útil para comprobar el contenido estático de un ZIM antes de permitir la ejecución de scripts. También es compatible con navegadores o frameworks antiguos que no pueden ejecutar Service Workers. El modo tiene limitaciones que significan que sólo se puede mostrar contenido estático, como el que se encuentra en los archivos de Wikipedia / WikiMedia y (por ahora) Stackexchange. Es una forma de insertar artículos extraídos del ZIM en el DOM (documento del navegador) inyectando el contenido en un iframe o pestaña. A continuación, utilizamos métodos DOM nativos, o a veces JQuery, para analizar el HTML del artículo e insertar los activos necesarios (imágenes, hojas de estilo, escuchadores de eventos para los hipervínculos, etc.). En los dispositivos antiguos, el recorrido del DOM puede ser lento, pero se compensa porque no extraemos ni ejecutamos activos JavaScript (lo que sería técnicamente extremadamente complicado). Como resultado, para los archivos WikiMedia este modo suele ser bastante rápido. Por otro lado, los ZIM que tienen una interfaz de usuario dinámica propietaria (como Gutenberg o TED Talks) sólo se admiten parcialmente en este modo: la interfaz de usuario no funciona, pero los artículos se pueden buscar y cargar desde la barra de búsqueda.",
```

Comment:
Could you change the second "scripts" in the first sentence to "activos"? It would read better like that.